File: programme/check-even-or-odd-string/CheckEvenOrOddString.java
// Write a programme to check even or odd string

import java.util.Scanner;

public class CheckEvenOrOddString {
  
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Input : ");
    String input = sc.nextLine();
    System.out.println("Output : " + checkEvenOrOddString(input) + " String");
    sc.close();
  }
  
  public static String checkEvenOrOddString(String input) {
    return (input.length() % 2 == 0) ? "Even" : "Odd";
  }
}

File: programme/check-even-or-odd-string/check-even-or-odd-string.py
# Write a programme to check even or odd string

given_string = input("Input your string : ")

print("\nInput  :", given_string)
print("Output : ", end="")
if (len(given_string) % 2 == 0):
    print("Even String")
else:
    print("Odd String")
